packages/aft/lib/src/commands/test_command.dart

```diff
@@ -0,0 +1,132 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:io';
+
+import 'package:aft/aft.dart';
+import 'package:aft/src/test_reports/test_report_folio.dart';
+import 'package:aft/src/test_reports/test_report_pass_fail.dart';
+import 'package:aft/src/test_reports/test_report_scored.dart';
+import 'package:aft/src/utils/emphasize_text.dart';
+
+/// Top-level command for running Flutter, Dart and integration tests.
+class TestCommand extends AmplifyCommand {
+  TestCommand(this.args);
+  List<String> args;
+
+  @override
+  String get description =>
+      'Top-level command for running Flutter, Dart and integration tests.';
+
+  @override
+  String get name => 'test';
+
+  bool _verbose = false;
+
+  @override
+  Future<void> run() async {
+    final testArgs = args.sublist(args.indexOf('test') + 1);
+    TestReportFolio? unitTestOutput;
+    TestReportFolio? integTestOutput;
+
+    if (testArgs.isEmpty) {
+      unitTestOutput = await UnitTestCommand().run();
+      integTestOutput = await IntegrationTestCommand().run();
+    }
+
+    if (testArgs.contains('unit')) {
+      unitTestOutput = await UnitTestCommand().run();
+    }
+
+    if (testArgs.contains('integ')) {
+      integTestOutput = await IntegrationTestCommand().run();
+    }
+
+    if (testArgs.contains('v')) {
+      _verbose = true;
+    }
```

Comment:
I'd prefer to merge https://github.com/aws-amplify/amplify-flutter/pull/1811 and use that method instead

Reply:
Cool - I approved #1762. I will probably merge it before you are back.